drivers/imx-enet.c

```diff
@@ -820,78 +1180,92 @@ static void enet_setLinkState(void *arg, int state)
 }
 
 
+#if ENET_SELFTEST
+#define _TP_DST     "dddddd"
+#define _TP_SRC     "ssssss"
 #define _TP_ETHTYPE "\x05\xDD" /* eth frame type 0x05DD is undefined */
 #define _TP_10DIG   "0123456789"
-#define TEST_PACKET "ddddddssssss" _TP_ETHTYPE \
+#define TEST_PACKET _TP_DST _TP_SRC _TP_ETHTYPE \
 	_TP_10DIG _TP_10DIG _TP_10DIG _TP_10DIG _TP_10DIG _TP_10DIG _TP_10DIG
-#define TEST_PACKET_LEN (sizeof((TEST_PACKET)) - 1)
-
+#define TEST_PACKET_LEN             (sizeof((TEST_PACKET)) - 1)
+#define ENET_PHYSELFTEST_RX_TIMEOUT (100 * 1000) /* TX -> RX takes ~4ms, wait for 100ms just to be sure */
```

Comment:
comment in the only place it's being used has better readability, please revert the change